### PR TITLE
[WIP] 917_KAM1

### PR DESCRIPTION
**Pull request by kameyer226**
## Changes

please describe your changes here :smiley:

## Checklist

- [ ] added tests, if applicable
- [ ] updated documentation, if applicable
- [ ] updated CHANGELOG.md
- [ ] tests pass

### CODE REVIEW

**Comment by kameyer226** [comment]
may require a test script

File: WrightTools/collection/_collection.py [hunk 1]
```diff
@@ -102,6 +102,17 @@ def create_collection(self, name="collection", position=None, **kwargs):
...
         return collection
 
+    def convert(self, units):
+        """Convert units of a collection.
+        Parameters
+        ----------
+        units: string
+            Units to convert to.
+        """
+        self.convert(units)
```
Comment (ksunden) [inline WrightTools/collection/_collection.py:112]: This will be infinite recursion right off the bat.

Comment (kameyer226) [reply WrightTools/collection/_collection.py:112]: Yes... guess this is a placeholder for further discussion.

**Comment by untzag** [comment]
@kameyer226 are you feeling stuck on this? I think the next step is to just try using it locally on your machine. Give it a shot, and let us know if you have problems that you can't figure out.

**Comment by kameyer226** [comment]
No, not yet...just waited to see what everyone had to say before making some tweaks.

**Pull request closed without merging**